src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -909,5 +909,10 @@ public enum SyntaxKind : ushort
         InterpolatedRawStringEndToken = 9074,               // """ (preceding whitespace and newline are included in the Text for this token)
 
         ScopedType = 9075,
+
+        CollectionCreationExpression = 9076,
```

Comment:
Consider putting these out at 9500+ with a prototype comment until merge into main, so you don't have to deal with merge conflicts.

Reply:
Enh.  The merge conflicts still happen (since someone still edits thsi same location) And all the tests still need to be updated :'(